### PR TITLE
imagej2: Add package

### PR DESCRIPTION
**Pull request by Kiri2002**
## Involved packages

 - imagej2

## Involved issue
Add new package
Close #

## Details

- [x] Tested in the local machine (largest 16G RAM) and it is passed without any issue
- [x] Provide New Package
- [ ] Fix the Packages
  - [ ] PKGBUILD
  - [ ] lilac.yaml
  - [ ] lilac.py
- [ ] Would like to continue to work with us

### CODE REVIEW

**Comment by Kiri2002** [comment]
[imagej2-2.15.0-0-x86_64.pkg.tar.zst-namcap.log](https://github.com/BioArchLinux/Packages/files/15143501/imagej2-2.15.0-0-x86_64.pkg.tar.zst-namcap.log)
[imagej2-2.15.0-0-x86_64-build.log](https://github.com/BioArchLinux/Packages/files/15143502/imagej2-2.15.0-0-x86_64-build.log)
[imagej2-2.15.0-0-x86_64-package.log](https://github.com/BioArchLinux/Packages/files/15143503/imagej2-2.15.0-0-x86_64-package.log)
[imagej2-2.15.0-0-x86_64-prepare.log](https://github.com/BioArchLinux/Packages/files/15143504/imagej2-2.15.0-0-x86_64-prepare.log)
[PKGBUILD-namcap.log](https://github.com/BioArchLinux/Packages/files/15143505/PKGBUILD-namcap.log)
```
imagej2 E: ELF file ('usr/share/imagej2/ImageJ-linux64') outside of a valid path.
imagej2 W: ELF file ('usr/share/imagej2/ImageJ-linux64') lacks FULL RELRO, check LDFLAGS.
imagej2 W: ELF file ('usr/share/imagej2/ImageJ-linux64') lacks PIE.
imagej2 W: ELF file ('usr/share/imagej2/ImageJ-linux64') lacks GNU_PROPERTY_X86_FEATURE_1_SHSTK.
imagej2 W: Unused shared library '/usr/lib/libdl.so.2' by file ('usr/share/imagej2/ImageJ-linux64')
imagej2 W: Unused shared library '/usr/lib/libpthread.so.0' by file ('usr/share/imagej2/ImageJ-linux64')
Checking imagej2-debug-2.15.0-0-x86_64.pkg.tar.zst
imagej2-debug W: Directory (usr/src/debug/imagej2) is empty
```
The issue of ELF is caused by upstream. It doesn't matter to build/run app.
I think it is acceptable, and lack of way to adress it.

**Comment by starsareintherose** [comment]
@sukanka @lilydjwg @hubutui @pekkarr 
Could you check recent changes? If it's okay, anyone of you can emerge, I'll go back 16 May, recently I am busy working on moving to Sydney

File: BioArchLinux/imagej2/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+# _pkgver=${pkgver//_/-}
+pkgrel=1
+pkgdesc='Open scientific N-dimensional image processing
+         https://doi.org/10.1186/s12859-017-1934-z'
+arch=('x86_64')
+url='https://imagej.net'
+license=('BSD-2-Clause')
+depends=(
+  'glibc'
+  'java-runtime=8'
```
Comment (starsareintherose) [inline BioArchLinux/imagej2/PKGBUILD:19]: I am not sure if it's okay

**Review by starsareintherose — COMMENTED** [review]
Others look fine to me

**Pull request merged**